store/indicators.js

```diff
@@ -0,0 +1,95 @@
+// this store manages NCAR 12km indicator data
+import _ from 'lodash'
+import { convertMmToInches, convertValueToFahrenheit } from '../utils/convert'
+import { getHttpError } from '../utils/http_errors'
+
+var convertTemperatureData = function (obj) {
```

Comment:
I see why/how this (and the other) functions here are sorta crafted to do the unit conversion for the indicators (in terms of the data structure).  But I'd suggest asking Craig to do a deep-dive on this with you to make sure that there isn't existing/related code in another endpoint -- like, the GIPL data conversion is in `utils/convert` (which actually probably needs to get tossed or updated in that PR).

Reply:
Looks like the conversion code in this file is using our existing unit conversion functions from `utils/convert.js` as-is and adding an extra layer recursion on top of them to crawl through data objects. So, I think this is a great first step and is not duplicating existing code. We'll probably want to see how to consolidate and clean up unit conversion code in the future, but that strikes me as best tackled as a separate chunk of work, already described in issue #512.